src/js/controller/controller.js

```diff
@@ -789,6 +791,10 @@ define([
             this.setCaptions = _view.setCaptions;
             this.next = _nextUp;
 
+            this.setConfig = function(newConfig) {
```

Comment:
More concisely expressed as `this.setConfig = (newConfig) => setConfig(_this, newConfig);`